src/Geometry/clipper.cs

```diff
@@ -4171,7 +4171,6 @@ private void FixupFirstLefts3(OutRec OldOutRec, OutRec NewOutRec)
 					//same as FixupFirstLefts1 but doesn't call Poly2ContainsPoly1()
 					foreach (OutRec outRec in m_PolyOuts)
 					{
-						OutRec firstLeft = ParseFirstLeft(outRec.FirstLeft);
```

Comment:
@mihakrajnc `clipper.cs` is taken from here (http://www.angusj.com/delphi/clipper.php) without modifications (except the namespace).